hazelcast/src/main/java/com/hazelcast/client/impl/ClientHeartbeatMonitor.java

```diff
@@ -68,13 +73,41 @@ public void start() {
 
     @Override
     public void run() {
+        cleanupEndpointsWithDeathConnections();
+
         String memberUuid = clientEngine.getThisUuid();
         for (ClientEndpoint ce : clientEndpointManager.getEndpoints()) {
             ClientEndpointImpl clientEndpoint = (ClientEndpointImpl) ce;
             monitor(memberUuid, clientEndpoint);
         }
     }
 
+    private void cleanupEndpointsWithDeathConnections() {
+        ArrayList<ClientEndpoint> endpointsWithDeadConnections = new ArrayList<ClientEndpoint>();
+        for (ClientEndpoint ce : clientEndpointManager.getEndpoints()) {
```

Comment:
Don't use abbreviations. If you want to write something short, just use endpoint or so. Abbreviations make the code harder to read.